Makefile

```diff
@@ -67,9 +68,11 @@ _save-refcache:
 
 check-links-all: check-links check-links-older check-links-external
 
```

Comment:
I think this is better done in the classic way, as dependency on specific generated file, not on phony target.